assets/src/edit-story/components/library/panes/media/media3p/media3pPane.js

```diff
@@ -127,13 +141,22 @@ function Media3pPane(props) {
       currentPaddingLeft - scrollbarWidth + 'px';
   }, [scrollbarWidth, refContainer]);
 
-  const { media } = useMedia3pForProvider('unsplash', ({ state }) => ({
-    media: state.media,
-  }));
+  /**
+   * Effectively performs a search, triggered at most every 500ms.
+   */
+  const [changeSearchTermDebounced] = useDebouncedCallback(() => {
```

Comment:
I thought that the search was only going to apply when the user presses enter.

Reply:
I've asked Sam for clarification, as the current upload tab searches automatically without enter.

Reply:
It's also an eng decision, if we're concerned about the qps sent to Unsplash / Coverr. This is a bigger concern for Coverr than Unsplash though, so we can decide on this later.

Reply:
Will remove autocompolete in a subsequent PR